### DISCUSSION TITLE
1.30.3

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#17202] [General]
# Improvements
- [#5533](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5533) Expose `reverse_direction` option for TuYa TS0601_cover (@Koenkk)
- [#5537](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5537) Add OTA support for Xiaomi `MCCGQ12LM` (@Otnow)
- [#5546](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5546) Expose `voc_index` for IKEA E2112 (@0ip)
- [#5547](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5547) Expose Develco AQSZB-110 `voc` as μg/m3 instead of ppb (@sjorge)
- [#5535](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5535) [#5556](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5556) Make thermostat `weekly_schedule` more user friendly (@sjorge)
- [#5543](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5543) Expose `switch_scene` action for LIDL FB21-001 (@martin-schlossarek)
- [#5557](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5557) Support `schedule_with_preheat` `programming_operation_mode` for Danfoss 014G2461 (@tbarnekov)
- [#471](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/471) Support additional features for YSRAI lights (@Koenkk)
- #16879 Support color temperature for GS BRHM8E27W70-I1 (@Koenkk)
- [#5586](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5586) Expose `battery_cover` for Xiaomi MCCGQ13LM (@DJTerentjev)
- [#5600](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5600) Support OTA for more Innr devices (@Koenkk)
- #17123 Support hosting the frontend on a Unix socket (@oddlama)
- [#5603](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5603) Expose `battery` for Viessmann ZK03840 (@ChrisChoke)
- [#5604](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5604) Expose `duration` for Woox R7051 (@MrAdam)
- #17144 Expose `voc_index` sensor measurements as such in Home Assistant  (@tkintscher)

# Fixes
- [#5414](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5414) Fix configure of same Yale locks failing (@Koenkk)
- [#5532](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5532) Fix TuYa TS0601_cover `_TZE200_r0jdjrvi` position reversed (@Koenkk)
- #15885 Fix Philips `929003526202_02` not detected as `929001953101` (@Koenkk)
- [#5534](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5534) Fix incorrect state for IKEA ICPSHC24-10EU-IL-1 and ICPSHC24-30EU-IL-1 when brightness is 1 (@Electry)
- #16873 [#5544](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5544) #16941 #17006 #17014 #17143 Disable unsupported `power_on_behavior` for more devices (@Koenkk, @Danit2)
- [#5539](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5539) Expose missing `pi_heating_demand` for Stelpro ST218 (@titilambert)
- [#5540](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5540) Fix `system_mode` of TuYa TS0601_thermostat_1 (@Delorien84)
- #16822 Fix device not controllable after interview (@Koenkk)
- #16871  Fix `occupancy` value of TuYa MIR-HE200-TY (@raphaabreu)
- [#5542](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5542) Fix missing `battery` expose for Salus PS600 (@mikeyyuen)
- [#5541](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5541) Fix configure failing for TS0121_plug (`_TZ3000_3ooaz3ng`) (@Enzokot)
- #16828 Fix old state send when long debounce time is used (@baze-)
- [#5548](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5548) Fix `pm25` `0` value not published (@0ip)
- [#5554](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5554) Fix X/Y color conversion (@TheJulianJES)
- #16057 Fix Home Assistant unit of measurement error for voc ppb sensors (@Koenkk)
- #16345 Fix persisted OTA of Xiaomi SRTS-A01 (@Koenkk)
- #16869 Fix crash when time write to Namron 4512737/4512738 fails (@Koenkk)
- #16162 Fix no `energy` published for OWON PC321 and CASAIA CCB432 (@Koenkk)
- [#5567](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5567) Fix GiEX QT06 behavior (@drakhart)
- [#16366](https://github.com/Koenkk/zigbee2mqtt/discussions/16366) Fix TuYa `_TZE200_9yapgbuv` not reporting data (@Koenkk)
- #16879 Fix `gas` detection not working for HEIMAN SGMHM-I1 (@Koenkk)
- [#5588](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5588) Fix for LIDL 14149505L/14149506L color handling (@AndreKR)
- [#680](https://github.com/Koenkk/zigbee-herdsman/pull/680) Fix end devices directly connected to the coordinator (ZStack adapter) becoming unreachable (@slugzero)
- #17074 Fix Lidl 14153905L color temperature range (@fs79)
- [#5597](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5597) Fix TuYa TS0601_thermostat calibration step value and max temperature range @xamadevp 
- #17086 Fix disappearing changes on device state update in frontend (@nurikk)
- #16863 Fix brightness decreases by 1 for TuYa TS110E_1gang_1 (@Koenkk)
- [#5599](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5599) Fix Bosch BTH-RA becoming unresponsive (@DerDreschner)
- #16911 Fix color control of Lidl 14149505L/14149506L not working (@Koenkk)
- [#686](https://github.com/Koenkk/zigbee-herdsman/issues/686) Fix command sending when cluster has non-unique id (@Koenkk)

# New supported devices
This release adds support for 30 devices:
- `33850000` Philips Philips Hue white ambiance pillar spot (1 spot) (@danruf)
- `3418031P6` Philips Hue white ambiance Adore GU10 with Bluetooth (3 spots) (@danruf)
- `4512747` Namron Curtain motor controller (@thoj)
- `915005916901` Philips Philips Hue white ambiance pillar double spot (2 spot) with Bluetooth (@danruf)
- `915005988501` Philips Play gradient light tube large (@chrisdrackett)
- `929002376901` Philips Hue Iris white (@GoldbergAlexander)
- `9290030521` Philips Hue white G125 B22 LED bulb filament giant globe (@sandert98)
- `9290030523` Philips Hue white filament Edison ST72 LED (@gdp2000)
- `929003074701` Philips Hue Xamento White and Color Ambiance GU10 (chrome) (@Koenkk)
- `AU-A1GSZ9B/27` Aurora Lighting AOne 9W smart GLS B22 (@nez-nix)
- `AUT000069` AutomatOn Underfloor heating controller - 5 zones (@Devligue)
- `BSD-2` Bosch Smoke alarm detector (@l154)
- `BWA-1` Bosch Zigbee smart water leak detector (@l154)
- `E2123` IKEA SYMFONISK sound remote gen2 (@ilueckel)
- `GL-G-004P` Gledopto Zigbee 7W garden light Pro RGB+CCT (@Javene)
- `HGZB-4S` Nue / 3A Smart light switch - 4 gang v2.0 (@chris-chensy)
- `HS2RNL` HEIMAN Smart repeater & night light (@FifouLeFilou)
- `IH-K663` TuYa Smart button (@kirovilya)
- `KK-BS-J01W` Konke Occupancy sensor (@kirovilya)
- `KK-DS-J01W` Konke Contact sensor (@kirovilya)
- `KK-TQ-J01W` Konke Smart 4 key scene switch (@kirovilya)
- `KK-WA-J01W` Konke Water detector (@kirovilya)
- `LED2102G3` IKEA TRADFRI bulb E27 WW 470 lumen, wireless dimmable warm white/globe clear (@hhampush)
- `QBKG29LM` Xiaomi Aqara smart wall switch H1 EU (no neutral, triple rocker) (@samuelthng)
- `ROB_200-060-0` ROBB Zigbee LED driver (@Koenkk)
- `S900W-ZG` HZC Water leak sensor (@BoneheadFraggle)
- `SA-028/SA-029` SONOFF Smart Plug (@mikedm139)
- `UEHK2AZ0` Universal Electronics Inc Xfinity security keypad (@bschrenk)
- `WSP403` OWON Smart plug (@Undef-a)
- `ZiPulses` LiXee Lixee ZiPulses (@ralmn)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.30.3'>1.30.3</a>.</em>

**Comment by cloudbr34k84** [comment]
who's game to update first.. lol

**Comment by swingstate** [comment]
I upgraded on Homeassistant and the add-on now fails to start. 

Log: 

`[16:35:48] INFO: Preparing to start...
[16:35:48] INFO: Socat not enabled
[16:35:51] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-04-01 16:35:58: Logging to console and directory: '/config/zigbee2mqtt/log/2023-04-01.16-35-58' filename: log.txt
Zigbee2MQTT:info  2023-04-01 16:35:58: Starting Zigbee2MQTT version 1.30.3 (commit #unknown)
Zigbee2MQTT:info  2023-04-01 16:35:58: Starting zigbee-herdsman (0.14.103)
Zigbee2MQTT:error 2023-04-01 16:36:19: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-04-01 16:36:19: Failed to start zigbee
Zigbee2MQTT:error 2023-04-01 16:36:19: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-04-01 16:36:19: Exiting...`

Is there a way to troubleshoot further?

**Reply by cloudbr34k84** [reply]
Do a full reboot and see if that helps??

**Reply by swingstate** [reply]
done that, add-on keeps restarting now.

**Comment by swingstate** [comment]
didnt change anything related to the adapter, just did the upgrade. Log points towards the adapter. 

info  2023-04-01 18:40:42: Logging to console and directory: '/config/zigbee2mqtt/log/2023-04-01.18-40-42' filename: log.txt
info  2023-04-01 18:40:42: Starting Zigbee2MQTT version 1.30.3 (commit #unknown)
info  2023-04-01 18:40:42: Starting zigbee-herdsman (0.14.103)
error 2023-04-01 18:41:03: Error while starting zigbee-herdsman
error 2023-04-01 18:41:03: Failed to start zigbee
error 2023-04-01 18:41:03: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
error 2023-04-01 18:41:03: Exiting...
error 2023-04-01 18:41:03: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:103:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:107:5)

**Reply by swingstate** [reply]
reinstalled the add-on, backed the config and started with a fresh config.

Tried to enter adapter port and MQTT server but get following error message, which seem to have appeared also with older releases. 

Fehler beim Speichern der Add-on Konfiguration, Invalid dict for option 'mqtt' in Zigbee2MQTT (45df7312_zigbee2mqtt). Got {'data_path': '/config/zigbee2mqtt', 'socat': {'enabled': False, 'master': 'pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777', 'slave': 'tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5', 'options': '-d -d', 'log': False}, 'mqtt': 'mqtt://core-mosquitto:1883', 'serial': '/dev/ttyAMA0'}

hardware info says: 

DEVLINKS: /dev/serial0
    DEVNAME: /dev/ttyAMA0
    DEVPATH: /devices/platform/soc/fe201000.serial/tty/ttyAMA0
    MAJOR: "204"
    MINOR: "64"
    SUBSYSTEM: tty
    TAGS: ':systemd:'
    USEC_INITIALIZED: "2729638"
  by_id: null
  children: []
  dev_path: /dev/ttyAMA0
  name: ttyAMA0
  subsystem: tty
  sysfs: /sys/devices/platform/soc/fe201000.serial/tty/ttyAMA0

any hint?

**Reply by swingstate** [reply]
I had to hard restart the Homeassistant device and did a rollback to the last Z2M release. Things are working now.

**Reply by joshuavaneck** [reply]
Edit: and it crashes again

I had exact same issues, found your post and replicated it..
Now z2m is working again! Thnx!

**Reply by belmondo2001** [reply]
I have also great Problems with this Release.
How can i rollback to 1.30.2 ?

**Reply by swingstate** [reply]
You can restore from the backups

**Reply by Koenkk** [reply]
If you have issue, please provide the herdsman debug logs.

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

**Comment by belmondo2001** [comment]
Here the Debug Log:

zigbee-herdsman:adapter:ezsp:ezsp Got EzspValueId.VALUE_VERSION_INFO = �� +9ms
  zigbee-herdsman:adapter:ezsp:driv EmberZNet version: 7.1.4.0 build 389 +101ms
  zigbee-herdsman:adapter:ezsp:ezsp ==> networkInit: null +17ms
  zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"networkInit","_id_":23,"_isRequest_":true} +4ms
  zigbee-herdsman:adapter:ezsp:uart --> DATA (7,7,0): 1700011700 +31ms
  zigbee-herdsman:adapter:ezsp:uart --> [775521a9432a0ae87e] +59ms
  zigbee-herdsman:adapter:ezsp:uart -?- waiting (0) +10ms
  zigbee-herdsman:adapter:ezsp:uart <-- [7055a5a9432a15b94c7e0055b1a94d2a8528c97e] +95ms
  zigbee-herdsman:adapter:ezsp:uart <-- DATA (7,0,0): 7055a5a9432a15b94c7e +23ms
  zigbee-herdsman:adapter:ezsp:uart --> ACK  (0) +4ms
  zigbee-herdsman:adapter:ezsp:uart --> [8070787e] +36ms
  zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 178401170000 +54ms
  zigbee-herdsman:adapter:ezsp:ezsp <== 0x17: {"_cls_":"networkInit","_id_":23,"_isRequest_":false,"status":0} +4ms
  zigbee-herdsman:adapter:ezsp:uart <-- DATA (0,0,0): 0055b1a94d2a8528c97e +25ms
  zigbee-herdsman:adapter:ezsp:uart --> ACK  (1) +4ms
  zigbee-herdsman:adapter:ezsp:uart --> [8160597e] +29ms
  zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 179001190090 +24ms
  zigbee-herdsman:adapter:ezsp:ezsp <== 0x19: {"_cls_":"stackStatusHandler","_id_":25,"_isRequest_":false,"status":144} +4ms
  zigbee-herdsman:adapter:ezsp:uart -+- waiting (0) success +19ms
  zigbee-herdsman:adapter:ezsp:ezsp network init result:  {"_cls_":"networkInit","_id_":23,"_isRequest_":false,"status":0} +9ms
  zigbee-herdsman:adapter:ezsp:ezsp ==> getNetworkParameters: null +3ms
  zigbee-herdsman:adapter:ezsp:ezsp ==> {"_cls_":"getNetworkParameters","_id_":40,"_isRequest_":true} +5ms
  zigbee-herdsman:adapter:ezsp:uart --> DATA (0,1,0): 1800012800 +16ms
  zigbee-herdsman:adapter:ezsp:uart --> [015a21a97c2ab2c77e] +37ms
  zigbee-herdsman:adapter:ezsp:uart -?- waiting (1) +11ms
  zigbee-herdsman:adapter:ezsp:uart <-- [1055b1a9902a7d38b2590b977e2055b1a9902a7d38b259ff957e3055b1a9902a7d38b259ac6b7e4055b1a9902a7d38b25907b07e] +95ms
  zigbee-herdsman:adapter:ezsp:uart <-- DATA (1,0,0): 1055b1a9902a18b2590b977e +17ms
  zigbee-herdsman:adapter:ezsp:uart --> ACK  (2) +3ms
  zigbee-herdsman:adapter:ezsp:uart --> [82503a7e] +29ms
  zigbee-herdsman:adapter:ezsp:uart Unexpected packet sequence 0 | 1 +9ms
  zigbee-herdsman:adapter:ezsp:ezsp <== Frame: 179001c4000d0000 +49ms
  zigbee-herdsman:adapter:ezsp:erro Frame changeSourceRouteHandler parsing error: RangeError: Attempt to access memory outside buffer bounds
  zigbee-herdsman:adapter:ezsp:erro     at new NodeError (node:internal/errors:387:5)
  zigbee-herdsman:adapter:ezsp:erro     at boundsError (node:internal/buffer:84:11)
  zigbee-herdsman:adapter:ezsp:erro     at Buffer.readUInt16LE (node:internal/buffer:243:5)
  zigbee-herdsman:adapter:ezsp:erro     at Buffer.readUIntLE (node:internal/buffer:180:17)
  zigbee-herdsman:adapter:ezsp:erro     at Function.deserialize (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/types/basic.ts:19:67)
  zigbee-herdsman:adapter:ezsp:erro     at new EZSPFrameData (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:96:54)
  zigbee-herdsman:adapter:ezsp:erro     at /opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:70:23
  zigbee-herdsman:adapter:ezsp:erro     at Array.every (<anonymous>)
  zigbee-herdsman:adapter:ezsp:erro     at Function.createFrame (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:68:15)
  zigbee-herdsman:adapter:ezsp:erro     at Ezsp.onFrameReceived (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:320:35) +0ms

**Comment by kirovilya** [comment]
@belmondo2001 this is not a critical bug - it is handled correctly and does not crash the system.

**Comment by bacobas** [comment]
@Koenkk  can you give us a PAYMENT  product support  ? 
We are currently using as coordinator: (Zigbee2mqtt on RaspberryPI) + (Sonoff harware Texas Chipset inside)

In some case we need to clear DATA directoy on Zigbee2mqtt device and RE Flash the Coordinator Chip : inothercase :  we can not re associate a determinate "Zigbee Roruterdevice"  or "Zigbee EndDevice".  

In some cases Although our device sends the Unbind command to the coordinator, (and from our device we verify that it is no longer connected to the coordinator)  On the Zigbee2mqtt platform it still appears in the devices and does not disappear... 

What could be the problems ?

We have developed a Zigbee product and actually we use your Zigbee2mqtt sevice to carry out the zigbee device data to our backend application ( Which is listening on the mqtt broker).

Can you support us  ?
BR

**Comment by terry389** [comment]
I have been having a similar problem. Everything has been working well for the last 6 months. Then last week after a update I haven't been able to get into Zigbee2mqtt. I have tried a few things including rebooting and restoring backups. Below is the Zigbee2MQTT debug log file and configuration file. Can anyone see what a fix would be to enable me to get back into Zigbee2MQTT UI.
Thanks

[07:52:19] INFO: Preparing to start...
[07:52:20] INFO: Socat not enabled
[07:52:20] INFO: Zigbee Herdsman debug logging enabled
[07:52:22] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:debug 2023-05-02 07:52:28: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2023-05-02 07:52:28: Logging to console and directory: '/config/zigbee2mqtt/log/2023-05-02.07-52-28' filename: log.txt
Zigbee2MQTT:debug 2023-05-02 07:52:28: Removing old log directory '/config/zigbee2mqtt/log/2023-05-02.07-32-00'
Zigbee2MQTT:info  2023-05-02 07:52:28: Starting Zigbee2MQTT version 1.30.4 (commit #unknown)
Zigbee2MQTT:info  2023-05-02 07:52:28: Starting zigbee-herdsman (0.14.111)
Zigbee2MQTT:debug 2023-05-02 07:52:28: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"adapter":"ezsp","path":"/dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20220713142810-if00"}}'
2023-05-01T21:52:29.264Z zigbee-herdsman:adapter Failed to validate path: 'Error: spawn udevadm ENOENT'
2023-05-01T21:52:29.268Z zigbee-herdsman:controller:log Starting with options '{"network":{"networkKeyDistribute":false,"networkKey":[1,3,5,7,9,11,13,15,0,2,4,6,8,10,12,13],"panID":6754,"extendedPanID":[221,221,221,221,221,221,221,221],"channelList":[11]},"serialPort":{"path":"/dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20220713142810-if00","adapter":"ezsp"},"databasePath":"/config/zigbee2mqtt/database.db","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","backupPath":"/config/zigbee2mqtt/coordinator_backup.json","adapter":{"disableLED":false,"concurrent":null,"delay":null}}'
2023-05-01T21:52:29.273Z zigbee-herdsman:adapter:ezsp:uart Opening SerialPort with {"path":"/dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20220713142810-if00","baudRate":115200,"rtscts":false,"autoOpen":false}
2023-05-01T21:52:29.295Z zigbee-herdsman:adapter:ezsp:uart Serialport opened
2023-05-01T21:52:29.296Z zigbee-herdsman:adapter:ezsp:uart Uart reseting
2023-05-01T21:52:29.298Z zigbee-herdsman:adapter:ezsp:uart --> Write reset
2023-05-01T21:52:29.301Z zigbee-herdsman:adapter:ezsp:uart --> [1ac038bc7e]
2023-05-01T21:52:29.305Z zigbee-herdsman:adapter:ezsp:uart -?- waiting reset
2023-05-01T21:52:39.306Z zigbee-herdsman:adapter:ezsp:uart --> Error: Error: {"sequence":-1} after 10000ms
2023-05-01T21:52:39.307Z zigbee-herdsman:adapter:ezsp:driv Reset connection. Try 0
2023-05-01T21:52:39.307Z zigbee-herdsman:adapter:ezsp:driv Stop driver
2023-05-01T21:52:39.310Z zigbee-herdsman:adapter:ezsp:ezsp Stop ezsp
2023-05-01T21:52:39.311Z zigbee-herdsman:adapter:ezsp:ezsp Close ezsp
2023-05-01T21:52:39.312Z zigbee-herdsman:adapter:ezsp:driv Close driver
Error: Reset error: Error: {"sequence":-1} after 10000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:299:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)


This is my Zigbee2MQTT Configuration File

data_path: /config/zigbee2mqtt
socat:
  enabled: false
  master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
  slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
  options: "-d -d"
  log: false
mqtt:
  server: mqtt://core-mosquitto:1883
  user: mqttuser
  password:
serial:
  port: >-
    /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20220713142810-if00
  adapter: ezsp
frontend: true
advanced: null
zigbee_herdsman_debug: true

**Reply by svkashaev** [reply]
Solved it by manually deactivating Zigbee Home Automation Integration which I have installed prior to Z2M add-on. Using 1.30.4 version.

**Reply by bacobas** [reply]
hi @svkashaev ;  how do you deactivate the Zigbee Home Automation Integration ?
What do you mean that you  had install Zigbee Home Automation "prior" to Z2M add-on ?
Thanks a lot.
BR

**Reply by svkashaev** [reply]
Settings -> Devices & Services -> Integrations. I Installed ZHA when I was trying to test my Moes curtain switch module. I just forgot that there is better way (Z2M) to manage zigbee devices than ZHA. Was trying to solve the issue for couple days, it was so predictably easy.
Deactivated ZHA, installed Z2M and everything works just fine.
![image](https://user-images.githubusercontent.com/80834190/235885670-c02397c3-fa4f-480c-afa0-e0786bebcae7.png)

**Reply by bacobas** [reply]
May be a misunderstanding:  We use zigbee2mqtt (Ver.1.30.3 commit: 24c6b2e) platform with coordinator module Sonoff based on texas chip inside . 
On zigbee2mqtt i can not find "Devices & Services -> Integrations."..   :-(

**Reply by terry389** [reply]
Thanks. That worked for me. I can now access ZigBee2MQTT and I lost nothing.

**Reply by bacobas** [reply]
excuse me but : do you use only Zigbee2mqtt interface? or you use a solution manager like Home assistant ?
